I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 792, in script
    jump reve1_soin_enfant
ScriptError: could not find label 'reve1_soin_enfant'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 792, in script
    jump reve1_soin_enfant
  File "F:\Utilitaires\renpy-7.3.5-sdk\renpy\ast.py", line 1697, in execute
    rv = renpy.game.script.lookup(target)
  File "F:\Utilitaires\renpy-7.3.5-sdk\renpy\script.py", line 898, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'reve1_soin_enfant'.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
EthiQuest 1.0
Sun May 17 21:18:51 2020
